### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by qwandor**

### CODE REVIEW

File: bluez-async/src/macaddress.rs [hunk 1]
```diff
@@ -132,4 +132,19 @@ mod tests {
...
     }
+
+    #[test]
+    fn format() {
+        let mac_address = MacAddress([0x12, 0x34, 0x56, 0xab, 0xcd, 0xef]);
+
+        assert_eq!(format!("{}", mac_address), "12:34:56:AB:CD:EF".to_string());
+        assert_eq!(
+            format!("{:X}", mac_address),
+            "12:34:56:AB:CD:EF".to_string()
```
Comment (alsuren) [inline bluez-async/src/macaddress.rs:143]: I don't think you need to .to_string() here.

File: bluez-async/src/service.rs [hunk 2]
```diff
@@ -68,4 +68,13 @@ mod tests {
         let service_id = ServiceId::new("/org/bluez/hci0/dev_11_22_33_44_55_66/service0022");
         assert_eq!(service_id.device(), device_id);
     }
+
+    #[test]
+    fn to_string() {
+        let service_id = ServiceId::new("/org/bluez/hci0/dev_11_22_33_44_55_66/service0022");
+        assert_eq!(
+            service_id.to_string(),
+            "hci0/dev_11_22_33_44_55_66/service0022".to_string()
```
Comment (alsuren) [inline bluez-async/src/service.rs:77]: Oh here

**Review by alsuren — APPROVED** [review]

File: bluez-async/src/macaddress.rs (hunk 1 above)
Comment (qwandor) [reply bluez-async/src/macaddress.rs]: Done.

File: bluez-async/src/service.rs (hunk 2 above)
Comment (qwandor) [reply bluez-async/src/service.rs]: Done.

**Pull request merged**